azure/durable_functions/models/OrchestrationRuntimeStatus.py

```diff
@@ -27,3 +27,6 @@ class OrchestrationRuntimeStatus(Enum):
 
     Pending = 'Pending'
     """The orchestration instance has been scheduled but has not yet started running."""
+
+    Suspended = 'Suspended'
+    """The orchestration instance has been paused."""
```

Comment:
I am not sure if I should add this new status. Since I checked all the references, I didn't find a logic that needs to add suspended status. Despite of this I think should better still add this?